.travis.yml

```diff
@@ -42,8 +43,12 @@ install:
 
 script: cd ../tests; yarn; yarn test-app-$IMAGE_NAME
 
+after_failure:
+  script: cd $TRAVIS_BUILD_DIR && ./commit_screenshots.sh
```

Comment:
So, if I understand correctly, to take advantage of `commit_screenshots.sh`, we would need to create a "bugs" branch in this repo, and then the script will be able to commit any error screenshot generated during CI to that branch?

Reply:
Correct